services/application/src/actions/load-context.js

```diff
@@ -18,6 +18,7 @@ module.exports = async ({ applicationId, email, ipAddress } = {}) => {
   const accessLevelIds = [];
   const teamQuery = {
     applicationId,
+    active: true,
```

Comment:
Set this to `${ ne: false }` and then it will immediately work without data migration